.rubocop.yml

```diff
@@ -1,25 +1,67 @@
 AllCops:
-  TargetRubyVersion: 2.1
+  TargetRubyVersion: 2.4.1
+  DisplayCopNames: true
   Exclude:
-    - 'db/migrate/**'
-    - 'db/schema.rb'
+    - 'db/**/*'
+    - 'node_modules/**/*'
     - 'vendor/**/*'
 
-Lint/AssignmentInCondition:
+# Disable documentations, because it's all clear w/o it *trollface*
```

Comment:
Yeah, but it's forcing to have them, even when it's not needed and everything is quite obviously. It would be much better to leave it on the programmer’s conscience.

Reply:
@sudakov-eugene this project is all about forcing myself to write some clean and well-documented code. So you can just add `# TODO: turn it off later` here.